bin/filter.rb

```diff
@@ -56,6 +56,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
   doc.css("strong:nth-child(1)").each do |node|
     id = node.content.downcase.gsub(/[^a-z0-9-]+/, '-').gsub(/^-|-$/, '')
     node["id"] = id if id.length > 1
+
+    if (format_for_frontpage == false)
```

Comment:
`if !format_for_frontpage`

Reply:
Oh right, we said there was a default for `format_for_frontpage` to `false`